### PR TITLE
Update QuickStartIT.java

### PR DESCRIPTION
**Pull request by lesv**
Fixes: #2715
Reviewing QuickstartIT, it:

* creates a bucket
* Writes a list of assets to the bucket
* wants to see that bucket in the list of assets
* deletes the bucket.

It should instead:

* Write to a well known bucket w/ gs:/well-known-bucket/UUID/my-assets.txt
* See if well-known-bucket is in my-assets.txt
This way there isn't a hysteresis problem on creating and knowing about buckets.

- [x] `pom.xml` parent set to latest `shared-configuration`
- [x] Appropriate changes to README are included in PR
- [x] API's need to be enabled to test (tell us)
- [x] Environment Variables need to be set (ask us to set them)
- [x] Tests pass (`mvn -P lint clean verify`)
  * (Note- `Checkstyle` passing is required; `Spotbugs`, `ErrorProne`, `PMD`, etc. `ERROR`'s are advisory only)

### CODE REVIEW

**Comment by lesv** [comment]
Looks like someone tried to fix things a different way.  Need to figure out what's going on.

**Comment by lesv** [comment]
I've resolved the merge conflict.

File: asset/cloud-client/src/test/java/com/example/asset/QuickStartIT.java [hunk 1]
```diff
@@ -45,33 +43,29 @@
 @RunWith(JUnit4.class)
 @SuppressWarnings("checkstyle:abbreviationaswordinname")
 public class QuickStartIT {
-  private static final String bucketName = UUID.randomUUID().toString();
+  private static final String bucketName = "java-docs-samples-testing";
```
Comment (kurtisvg) [inline asset/cloud-client/src/test/java/com/example/asset/QuickStartIT.java:46]: Can we use an environment var for this?

Comment (elharo) [reply asset/cloud-client/src/test/java/com/example/asset/QuickStartIT.java:46]: Can anyone who is not a googler still run this integration test?

**Pull request merged**